database/mysql/sellers.js

```diff
@@ -0,0 +1,54 @@
+const mysql = require('./index');
+
+module.exports = {
+  get: (param) => {
+    const q = 'SELECT * FROM seller WHERE id = ?';
+    return new Promise((resolve, reject) => {
+      mysql.query(q, [param], (err, results) => {
+        if (err) {
+          reject(err);
+        } else {
+          resolve(results);
+        }
+      });
+    });
+  },
+  create: (param) => {
+    console.log(param);
+    const q = 'INSERT INTO seller SET ?';
+    return new Promise((resolve, reject) => {
+      mysql.query(q, param, (err, results) => {
```

Comment:
Shouldn't param be `[param]` for the value placeholder assuming it's more than one column of data?

Reply:
@wontkim Thanks for the review.  
You would need an array for parameters if it's for a get query.  For inserting data, the param should be an {} object.